frontend/src/assets/css/variables.scss

```diff
@@ -13,4 +13,9 @@
     --checkbox-disabled: #f7f8fc;
     --checkbox-checked: #274f8d;
     --checkbox-disabled-checked: #d5dce8;
+    --header-background: #ffffff;
```

Comment:
you don't need to duplicate color. You can create just 1 variable for white color